File: android_libraries/QR_With_Zxing/app/src/main/java/w2022v9o12/simple/qr_with_zxing/MainActivity.java
package w2022v9o12.simple.qr_with_zxing;

import androidx.appcompat.app.AppCompatActivity;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import w2022v9o12.simple.qr_with_zxing.databinding.ActivityMainBinding;
import w2022v9o12.simple.qr_with_zxing.databinding.ActivityScanQrBinding;
import w2022v9o12.simple.qr_with_zxing.qr_scan.ScanQrActivity;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        //QR Code 만들기
        binding.makeQRbtn.setOnClickListener(view -> goToOtherActivity(MakeQrActivity.class));

        //QR 스캔 화면으로 이동하기
        binding.scanQRbtn.setOnClickListener(view -> goToOtherActivity(ScanQrActivity.class));
    }

    //다른 Activity로 넘어가기
    private void goToOtherActivity(Class activity) {
        Intent intent = new Intent(this, activity);
        startActivity(intent);
    }

}

File: CameraX/CameraX_4_OCR_TextGraphic/app/src/main/java/w2022v9o12/simple/camerax_analysis/MainActivity.kt
package w2022v9o12.simple.camerax_analysis


import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import w2022v9o12.simple.camerax_analysis.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var viewBinding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        viewBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(viewBinding.root)

        viewBinding.imageAnalysisButton.setOnClickListener{
            val intent = Intent(this, ImageAnalysisActivity::class.java)
            startActivity(intent)
        }

        viewBinding.imageCaptureButton.setOnClickListener{
            val intent = Intent(this, ImageCaptureActivity::class.java)
            startActivity(intent)
        }
    }
}